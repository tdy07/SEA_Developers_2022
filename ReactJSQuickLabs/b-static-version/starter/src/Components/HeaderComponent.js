import React from 'react'
import Logo from './images/qalogo.svg'

function HeaderComponent() {
  return (
    <header>
        <nav className='navbar'>
            <a href='https://www.qa.com'>
            <img src={Logo} alt='QA Ltd' width='100'></img>
            </a>
            <a href='/' className='mr-auto'>
            <h1>Todo App</h1>
            </a>
        </nav>
    </header>
  )
}

export default HeaderComponent