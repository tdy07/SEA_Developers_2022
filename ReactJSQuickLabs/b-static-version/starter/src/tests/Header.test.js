import React from "react"
import { create } from "react-test-renderer"
import HeaderComponent from "../Components/HeaderComponent"

test('Header matches snapshot', () => {
    const header = create(<HeaderComponent/>)
    expect(header.toJSON()).toMatchSnapshot()
})