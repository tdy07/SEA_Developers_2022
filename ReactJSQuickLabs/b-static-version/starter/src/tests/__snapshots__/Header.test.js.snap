// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<header>
  <nav
    className="navbar"
  >
    <a
      href="https://www.qa.com"
    >
      <img
        alt="QA Ltd"
        src="qalogo.svg"
        width="100"
      />
    </a>
    <a
      className="mr-auto"
      href="/"
    >
      <h1>
        Todo App
      </h1>
    </a>
  </nav>
</header>
`;
